readme.md

```diff
@@ -43,6 +43,10 @@ Returns a `boolean` for whether it's a [`stream.Duplex`](https://nodejs.org/api/
 
 Returns a `boolean` for whether it's a [`stream.Transform`](https://nodejs.org/api/stream.html#stream_class_stream_transform).
 
+#### isStream.type(stream)
+
+Returns a `string` for which the [`type of stream`](https://nodejs.org/api/stream.html#stream_types_of_streams).
```

Comment:
Can you list the possible values? It's not immediately clear to the user what format it will be in.

Reply:
done f4b590b